### PR TITLE
fix: error details error is handled

### PR DESCRIPTION
**Pull request by palash-signoz**
error details error is handled

### CODE REVIEW

File: frontend/src/pages/ErrorDetails/index.tsx [hunk 1]
```diff
@@ -81,7 +84,18 @@ function ErrorDetails(): JSX.Element {
...
-		return <Typography>{data?.error || errorIdPayload?.error}</Typography>;
+		return (
+			<Typography>
+				{data?.error || errorIdPayload?.error || defaultError}
+			</Typography>
+		);
+	}
+
+	if (
+		(status === 'success' && data?.statusCode !== 200) ||
```
Comment (pranshuchittora) [inline frontend/src/pages/ErrorDetails/index.tsx:95]: Same here status code >= 400

Comment (palash-signoz) [reply frontend/src/pages/ErrorDetails/index.tsx:95]: not sure why we are going with >=400 ... if data is successfully fetched that’s only be applicable for 200 if not that's an error

Comment (pranshuchittora) [reply frontend/src/pages/ErrorDetails/index.tsx:95]: ok

Comment (palash-signoz) [reply frontend/src/pages/ErrorDetails/index.tsx:95]: is there any other use case ... that i am not not able to think here ?

**Comment by pranshuchittora** [comment]
@palash-signoz there are a lot of ifs in this file, some of them are self-explanatory but it would be great if you can add few comments explaining the condition


https://github.com/SigNoz/signoz/blob/6f48030ab91e81fb97b85eff1d802132b4d34128/frontend/src/pages/ErrorDetails/index.tsx

File: frontend/src/pages/ErrorDetails/index.tsx (hunk 1 above)
Comment (pranshuchittora) [reply frontend/src/pages/ErrorDetails/index.tsx:95]: Or any other redirect in 300 series _(with Axolo)_

Comment (palash-signoz) [reply frontend/src/pages/ErrorDetails/index.tsx:95]: cool will add _(with Axolo)_

**Review by pranshuchittora — APPROVED** [review]

**Pull request merged**